app/views/courses/TeacherClassesView.coffee

```diff
@@ -118,8 +122,9 @@ module.exports = class TeacherClassesView extends RootView
 
   afterRender: ->
     super()
-    @courseNagSubview = new CourseNagSubview()
-    @insertSubView(@courseNagSubview)
+    unless @courseNagSubview
+      @courseNagSubview = new CourseNagSubview()
+      @insertSubView(@courseNagSubview)
```

Comment:
I think the `insertSubView` line will need to happen after each render.

Reply:
Just insert without new doesn't work, and doing this on every render generates a bunch of duplicate prepaid fetches.  This seems to work fine, teacher quest stuff gets updated as data loads.